assets/js/vue/components/information-product/InformationProductCard.vue

```diff
@@ -22,6 +22,9 @@
         <div v-if="informationProduct.publisher">
           Publisher: {{ informationProduct.publisher | truncate(100) }}
         </div>
+        <div>
+          Created/Updated: {{ informationProduct.modificationTimeStamp }}
```

Comment:
I felt something like this is needed in the info product card, because it shows why the card is in the sort. Also, need to figure out the moment library to do date formatting both here and in the dataset published date field (same ugliness...in production like that).